libraries/src/HTML/Helpers/Bootstrap.php

```diff
@@ -528,11 +528,15 @@ public static function toast($selector = '.toast', $options = []) :void
 	 */
 	public static function framework($debug = null) :void
 	{
+		$wa = Factory::getApplication()
+			->getDocument()
+			->getWebAssetManager();
+
 		array_map(
-			function ($script) {
-				HTMLHelper::_('bootstrap.' . $script);
+			function ($script) use ($wa) {
+				$wa->useScript('bootstrap.' . $script);
 			},
-			static::$scripts
+			['alert', 'button', 'carousel', 'collapse', 'dropdown', 'modal', 'popover', 'scrollspy', 'tab', 'toast', 'tooltip']
```

Comment:
@Fedik you don't need `tooltip` as `poopover` and `tooltip` are bundled together
```suggestion
			['alert', 'button', 'carousel', 'collapse', 'dropdown', 'modal', 'popover', 'scrollspy', 'tab', 'toast']
```

Reply:
okay, I just copied old scripts :smile: